kong/router/atc.lua

```diff
@@ -666,14 +673,38 @@ function _M:exec(ctx)
 
   req_uri = strip_uri_args(req_uri)
 
+  local src_ip, src_port, dst_ip, dst_port
+  do
+    if self.fields["net.src.ip"] then
+      src_ip = var.remote_addr
+    end
+
+    if self.fields["net.src.port"] then
+      src_port = tonumber(var.remote_port, 10)
+    end
+
+    if self.fields["net.dst.ip"] then
+      dst_ip = var.server_addr
+    end
+
+    if self.fields["net.dst.port"] then
+      dst_port = tonumber(var.server_port, 10)
+    end
+  end
+
   -- cache lookup
 
   local cache_key = (req_method  or "") .. "|" ..
                     (req_uri     or "") .. "|" ..
                     (req_host    or "") .. "|" ..
                     (sni         or "") .. "|" ..
                     (headers_key or "") .. "|" ..
-                    (queries_key or "")
+                    (queries_key or "") .. "|" ..
```

Comment:
Where do we test this change?

Reply:
It seems that we have no test case for cache key.